config/default_configs/default_config.yml

```diff
@@ -51,7 +51,7 @@ topo_smoothing:
   value: false
 topography_damping_factor:
   help: "Factor by which smallest resolved length-scale is to be damped"
-  value: 512
+  value: 5
```

Comment:
This seems like a pretty large change. What value should we use in the AMIP simulation?

Reply:
The default value in the AMIP configuration initially was set up to match the outcome from the working Gaussian configuration in the coarse AMIP config. Ideally I'd like to use no more than 10-100x attenuation at the small scales if it's shown to be robust across all our configs. (The first iteration of this was perhaps overly conservative). This outcome of this PR is tested against the 16 element longrun build result for now (up to 60days). (Plots of the surface elevation for h_elem=16 are below as part of the doc assets. )

Reply:
Could you also try this in the nightly amip?

Reply:
https://buildkite.com/clima/climacoupler-coarse-nightly-amip/builds/136 @szy21 the corresponding build is here